client/CBitmapHandler.cpp

```diff
@@ -42,33 +42,36 @@ SDL_Surface * BitmapHandler::loadH3PCX(ui8 * pcx, size_t size)
 	SDL_Surface * ret;
 
 	Epcxformat format;
-	int it=0;
-
-	ui32 fSize = read_le_u32(pcx + it); it+=4;

```

Comment:
IMO this indicates different problem - `it+=4` should be part of `read_le_u32` - e.g. wrapped in some memory buffer class instead of manually advancing pointer for correct length each time you call this method.